config/dagmc.mk

```
@@ -1,3 +1,11 @@
+DAGMC_BUILDDIR := $(CARDINAL_DIR)/build/DAGMC
+DAGMC_INSTALL_DIR := $(CONTRIB_INSTALL_DIR)
+
+# By default, build dagmc
+BUILD_DAGMC ?= yes
```

```suggestion
BUILD_DAGMC ?= no
```
Default behaviour in Cardinal is not build with DAGMC unless the user specifies that they want CAD geometry support (due to the additional build burden).